src/libsyntax/feature_gate.rs

```diff
@@ -284,11 +284,7 @@ impl<'a, 'v> Visitor<'v> for PostExpansionVisitor<'a> {
         }
         match i.node {
             ast::ItemExternCrate(_) => {
```

Comment:
Can you add a message telling people what to do if they still have a `#[plugin]`-tagged extern crate?